ee/tabby-ui/components/load-more-indicator.tsx

```diff
@@ -0,0 +1,39 @@
+'use client'
+
+import React from 'react'
+import { useInView } from 'react-intersection-observer'
+
+import { cn } from '@/lib/utils'
+
+interface Props {
+  onLoading: () => void
+  className?: string
+}
+
+const LoadMoreIndicatorRender: React.FC<React.PropsWithChildren<Props>> = ({
+  onLoading,
+  children,
+  className
+}) => {
+  const { ref, inView } = useInView()
+  const [isLoaded, setIsLoaded] = React.useState(false)
+  React.useEffect(() => {
+    if (inView && !isLoaded) {
```

Comment:
Sorry I don't understand what is `isLoaded`, can you elaborate it?

Reply:
Good catch! There is a bug in this implementation, I have modified it